src/spoke/BalanceSheet.sol

```diff
@@ -231,7 +233,7 @@ contract BalanceSheet is Auth, Multicall, Recoverable, IBalanceSheet, IBalanceSh
             assetQueue.deposits,
             assetQueue.withdrawals,
             pricePoolPerAsset,
-            isSnapshot,
+            shareQueue.delta == 0 && shareQueue.queuedAssetCounter == assetCounter, //isSnapshot
```

Comment:
Q: Can you elaborate on this change? Seems like it slightly increases gas while also reducing readability

Reply:
Q: Why do we not need extra gas for submitting queued assets but need it for queued shares?

Reply:
> Can you elaborate on this change? Seems like it slightly increases gas while also reducing readability

Yes stack too deep issue 😢

> Q: Why do we not need extra gas for submitting queued assets but need it for queued shares?

If I'm right, both methods have `extraGasLimit`

Reply:
> If I'm right, both methods have extraGasLimit

You're right, sorry! Looked in the wrong row where the event emission happens.